src/core/base-layout/components/header/nav-item/nav-item.tsx

```diff
@@ -38,20 +39,23 @@ export const NavItem = ({ label, href, children }: NavItemProps) => {
     : pathname?.includes(href.replaceAll(ROUTES.HOME, ''));
   const linkHref = isHrefHome ? href : `/${href}`;
 
-  const handleConfirmKeyPress = (e: KeyboardEvent<HTMLButtonElement>) => {
-    if (e.code === 'Enter' || e.code === 'Space') {
-      e.preventDefault();
-      setDropdownOpen((prev) => !prev);
-    }
-  };
-
   const handleEscKeyPress = (e: KeyboardEvent<HTMLElement>) => {
     if (e.code === 'Escape') {
```

Comment:
Let's all keys' codes move to constant like `KEY_CODES = {ESCAPE: 'Escape', .....} You could add it here `src/core/const/index.ts`